spec/vc.md

```diff
@@ -0,0 +1,80 @@
+# W3C Verifiable Credentials v1.1
+
+The VC Data Model provides optionality for many of its properties. This means they can take on multiple concrete types, for example an [`issuer` property](https://www.w3.org/TR/vc-data-model/#issuer) can either be repreesnted as a JSON string representing a URI or a JSON object that must contain an `id` property. Additionally, the data model follows an [open world](https://www.w3.org/TR/vc-data-model/#extensibility) model for extensibility via the usage of [JSON-LD](https://www.w3.org/TR/json-ld11/). As a consequence, conformant Verifiable Credentials may contain properties that are not defined by the specification itself, but by [JSON-LD Contexts](https://www.w3.org/TR/json-ld11/#the-context).
+
+This optionality can be difficult to implement consistently across languages. As a consequence, this specification defines a strict subset of the VC Data Model v1.1 that supports the [plain JSON syntax](https://www.w3.org/TR/vc-data-model/#json), represented by the following table. As a utility JSON Schemas for [Verifiable Credentials](vc-11.json) and [Verifiable Presentations](vp-11.json) are provided to aid in the validation of conformant documents.
+
+## Verifiable Credential Data Model
+
+Following from [this data model](https://www.w3.org/TR/vc-data-model/#basic-concepts).
+
+| Property      | JSON Representation | Required | Notes          |
+| ------------- | ------------------- | -------- | -------------- |
+| `@context`    | Array of strings    | Yes      | Contexts defining the meaning of terms within the credential. Must include at least `"https://www.w3.org/2018/credentials/v1"`. |
+| `id`          | String              | Yes      | A URI representing a unique identifier for the credential. Recommended to be of form `urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5`. |
+| `type`        | Array of strings    | Yes      | Type(s) of the credential. Must include `VerifiableCredential`. |
+| `issuer`      | String              | Yes      | A DID representing a unique identifier for the entity that issued the credential. |
```

Comment:
will we accept outside vcs that are in this form?

```  
"issuer": {
    "id": "did:example:76e12ec712ebc6f1c221ebfeb1f",
    "name": "Example University"
  },
  ```
  
  and then convert the id internally to just a string?

Reply:
I need to revise this to be either...I don't think we can get away with supporting just one
the DMV uses an object, but for 90% of our cases we wont